src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfo.cs

```diff
@@ -200,10 +202,6 @@ private void DetermineSerializationCapabilities(JsonSerializerOptions options)
                     {
                         ShouldDeserialize = true;
                     }
-                    //else
-                    //{
-                    //    // todo: future feature that allows non-List types (e.g. from System.Collections.Immutable) to have converters.
-                    //}
                 }
                 //else if (HasSetter)
```

Comment:
dead code comment

Reply:
fyi @steveharter - I noticed similar dead code earlier. Not sure if you were using it to track some feature/up coming code path.

FWIW, I prefer not having commented out code anyway.